index.js

```diff
@@ -12,5 +12,10 @@ const tutorials = [
 ];
 
 const titleCased = () => {
-  return tutorials
-}
+  return tutorials.map(word => 
+  word.split(' ')
+  .map(word => word[0].toUpperCase() + word.slice(1)
+  )
+  .join(' ')
+);
+};
```

Comment:
This is great! It works! But it's a little messy which can make it harder to understand than needed. You could try using the VS Code extension called "Prettier" which auto-formats your code for you. Either way, make sure you take the time to clean up your syntax before submitting in the future. Thanks!
```suggestion
  return tutorials.map(word => 
    word.split(' ')
      .map(word => word[0].toUpperCase() + word.slice(1))
      .join(' ')
  );
};
```